doc/whats_new/v0.20.rst

```diff
@@ -1253,6 +1253,10 @@ Miscellaneous
   happens immediately (i.e., without a deprecation cycle).
   :issue:`11741` by `Olivier Grisel`_.
 
+- |Fix| Fixed a bug in validation.py where :func: `_num_samples()` would not properly
+  handle checking the shape of Dask DataFrames after an update to Dask. 
+  :issue:`12462` by :user:`zwmiller`
```

Comment:
Usually we'd state your real name too

Reply:
Got it. Sorry for the hand-holding needed.